source/standards/performance-testing.html.md.erb

```diff
@@ -0,0 +1,99 @@
+---
+title: Performance testing
+last_reviewed_on: 2018-10-31
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+Make sure your service performs the way users need it to. If your service does not perform as expected when there are extra loads it will impact your users.
+
+You should run performance tests to check how your service performs in certain situations. For example, during a [denial-of-service attack (DoS attack)](https://www.ncsc.gov.uk/guidance/denial-service-dos-guidance-collection).
+
+The Service Manual has more information about why you need to [test your service’s performance](https://www.gov.uk/service-manual/technology/test-your-services-performance) and how to start capacity planning and running performance tests.
+
+## Plan your tests
+
+Plan and scope your test whether you're testing the entire service or specific user journeys.
+
+### Estimate costs
+
+Estimate any additional infrastructure costs your tests will incur and share them with your service owner or product manager for approval.
+
+### Plan user journeys
+
+Test complete journeys that are representative of how your service is used in production. For example, if a user has to sign in or register, plan tests where some users sign in and others register.
+
+### Plan test scenarios
+
+A scenario is a hypothetical event when your service will be under heavy load. For example, every year the [G-Cloud framework on the Digital Marketplace](https://www.gov.uk/guidance/the-g-cloud-framework-on-the-digital-marketplace) opens for applications from suppliers for several weeks. This activity increases load on the [Digital Marketplace](https://www.digitalmarketplace.service.gov.uk/) up to the deadline.
+
+When most of its application infrastructure moved to the [GOV.UK PaaS](https://www.cloud.service.gov.uk/) the Digital Marketplace service needed to retest the service. Its developers and performance analysts queried Google Analytics for popular journeys, modelling normal and peak capacity needs for the previous year’s G-Cloud framework application period.
```

Comment:
Another thing that we did was to export historical router access logs from CloudWatch for the period of interest (the previous G-Cloud's iteration) and shove them into a locally running Kibana instance. This really helped to show when that volumes of traffic were.

Reply:
Good point! Forgot about logs :)